build/frappe-nginx/docker-entrypoint.sh

```diff
@@ -45,13 +45,24 @@ if [[ -z "$UPSTREAM_REAL_IP_HEADER" ]]; then
     export UPSTREAM_REAL_IP_HEADER="X-Forwarded-For"
 fi
 
+if [[ -z "$FRAPPE_SITE_NAME_HEADER" ]]; then
+    export FRAPPE_SITE_NAME_HEADER="\$host;"
```

Comment:
`;` semi-colon should be removed from both variable value, it already exists in template file.

it causes template to result into extra semi-colon in `default.conf` file

```
        proxy_set_header X-Frappe-Site-Name $host;;
        proxy_set_header Host $http_host;;
```

Note the extra semi-colon.